src/builder.spec.ts

```diff
@@ -79,12 +79,7 @@ describe('Builder', () => {
     }
 
     test('should fail if extract-i18n fails', async () => {
-        await architectHost.addBuilder('@angular-devkit/build-angular:extract-i18n', createBuilder(() => ({success: false}))); // dummy builder
-        // write dummy file, so that reading before extraction will not fail:
-        await fs.writeFile(MESSAGES_XLF_PATH, '<xliff version="2.0" xmlns="urn:oasis:names:tc:xliff:document:2.0" srcLang="de">\n' +
-            '  <file original="ng.template" id="ngi18n">\n' +
-            '  </file>\n' +
-            '</xliff>', 'utf8');
+        architectHost.addBuilder('@angular-devkit/build-angular:extract-i18n', createBuilder(() => ({success: false}))); // dummy builder
```

Comment:
My editor was complaining about awaiting a void method. It's also happening in other places in this file, but I don't know how you feel about sweeping changes so I only got this one (and the one in the new test).

Reply:
Not sure either ;)
Always pro "Boy Scout" - but like separate PRs as well..